proctor-common/src/main/java/com/indeed/proctor/common/ForceGroupsDefaultMode.java

```diff
@@ -15,6 +15,10 @@ public enum ForceGroupsDefaultMode {
      * It forces to assign no bucket. In most situations when you use AbstractGroups, it leads to using the fallback buckets.
      */
     FALLBACK("default_to_fallback"),
+    /**
+     * It forces to assign the minimum >0% bucket within the resolved allocation.
+     */
+    MIN_ACTIVE("default_to_min_active"),
```

Comment:
Could you add test cases for this in `TestFourceGroupsDefaultMode.java`?

Reply:
Added more tests.